<launch>
  <!-- panorama node -->
  <node name="Panorama" pkg="panorama" type="Panorama" output="screen" args="--v 0">

    <!-- Topics -->
    <param name="events_topic" value="/dvs/events" />
    <param name="camera_info_topic" value="/dvs/camera_info" />
    
    <!-- COMMON PARAMS -->
    <param name="contrast_measure" value="0" /> 
    <param name="frontend_event_sample_rate" value="1" /> 
    <param name="panorama_height" value="480" /> 
    <param name="panorama_width" value="3163" /> 
    <rosparam param="R_matrix">[1., 0., 0.,
                                0., -0.93033817, 0.36670272,
                                0., -0.36670272, -0.93033817]</rosparam>


  </node>

  <!-- Publish camera calibration -->

  <node name="dvs_cam_info_pub" pkg="panorama" type="publish_camera_info.py" output="screen"/>  

  <!-- Play rosbag -->
  <node pkg="rosbag" type="play" name="player" args="-d 1.0 -r 1.0 /home/dai/2025-04-03-16-20-51.bag" />


  <!-- Start rviz visualization with preset config -->
  	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find panorama)/config/count_pano.rviz" output="screen">
  </node>

</launch>
